experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -96,12 +96,28 @@ export interface HasLineage {
 
 export const HasLineage = Type.Object({ lineage: Type.Optional(Type.Array(LineageEvent)) });
 
+export interface IdRange {
+	id: ChangesetLocalId;
+	count: number;
+}
+
+export const IdRange = Type.Object({
+	id: ChangesetLocalIdSchema,
+	count: Type.Number(),
+});
+
 /**
  * @alpha
  */
-export interface CellId extends ChangeAtomId, HasLineage {}
+export interface CellId extends ChangeAtomId, HasLineage {
+	adjacentCells?: IdRange[];
+}
 
-export const CellId = Type.Composite([EncodedChangeAtomId, HasLineage]);
+export const CellId = Type.Composite([
+	EncodedChangeAtomId,
+	HasLineage,
+	Type.Object({ adjacentCells: Type.Optional(IdRange) }),
+]);
```

Comment:
It just occured to me that we probably should add `noAdditionalProps` to this. No worries if you don't want to do it as part of this PR though.
```suggestion
], noAdditionalProps);
```